docker-entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env sh
+
+if [ -n "$HTTP_BASIC_AUTH_USER" ] && [ -n "$HTTP_BASIC_AUTH_PASSWORD" ]
+then
+    echo "Generating password file: /home/mailhog/htpasswd"
+    echo "$HTTP_BASIC_AUTH_USER:$(MailHog bcrypt $HTTP_BASIC_AUTH_PASSWORD)" > /home/mailhog/htpasswd
+    MailHog -auth-file=/home/mailhog/htpasswd
+else
+    MailHog
```

Comment:
note that Mailhog will now run as child process of /bin/sh, and not be the container's primary process; use `exec Mailhog` instead to make it run as primary process

Reply:
Thanks, this will be useful for me :+1: